core/keyboard_nav/navigation.js

```diff
@@ -975,13 +977,15 @@ Blockly.navigation.toolboxOnAction_ = function(action) {
  */
 Blockly.navigation.enableKeyboardAccessibility = function() {
   Blockly.keyboardAccessibilityMode = true;
+  Blockly.navigation.focusWorkspace();
```

Comment:
is it an issue if this is called multiple times? Should it check the keyboardAccessibilityMode flag before making changes?

Reply:
Neither of these should be a problem if it is called more than once, but I can add the check just to be safe.